Lib/pathlib.py

```diff
@@ -30,6 +30,14 @@
 # Internals
 #
 
+# Reference for Windows paths can be found at
+# https://docs.microsoft.com/en-gb/windows/win32/fileio/naming-a-file
```

Comment:
```suggestion
# https://learn.microsoft.com/en-gb/windows/win32/fileio/naming-a-file .
```

"docs\.microsoft\.com" has recently migrated to "learn.microsoft.com". 

Note that omitting the language from the URL (e.g. "en-gb") would automatically redirect to the user's preferred language (e.g. "learn\.microsoft\.com/ru-ru/" or "learn\.microsoft\.com/en-us/").